docs/product vision/main.tex

```diff
@@ -65,28 +106,21 @@ \section{Unique Selling Points}
 \par
 So the key to what makes our software unique is that it's personalized and therefor extremely usable. It is specifically created for analysts to process the data from renal monitoring devices, hospital records and the renal monitoring website. The provided language for creating analysis can be kept concise and simple because the purpose is so specific, the graphical user interface can be kept clean because it desires not too many extended features, which any other data analysis tool already has. 
 \par 
-The company itself does not yet have software that compares with our software, but there are other groups working on the same assignment. This means we will have to outshine them in some way. Since the other groups have received the exact same assignment and the same description for the software's exact functionality requirements, we will need to create a software of high quality and usability to 'win'. We plan to do this by creating an easy and effective GUI and implementing as much automation in routine actions as possible. This will be achieved by using User Centered Design principles\cite{nielsen}.
-
-
+The company itself does not yet have software that compares with our software, but there are other groups working on the same assignment. This means we will have to outshine them in some way. Since the other groups have received the exact same assignment and the same description for the software's exact functionality requirements, we will need to create a software of high quality and usability to 'win'. We plan to do this by creating an easy and effective GUI and implementing as much automation in routine actions as possible. This will be achieved by using User Centered Design principles \cite{Nielsen}.
+%search literature on "web help" systems like mijnnierinzicht.nl
+%search relatable projects combining multiple data sources (to analyze users)
 \section{Time-frame}
-There's a time-frame of 10 weeks for the entire project. We aim to finish the product in week 9 so we will have the time to debug in week 10. Apart from the debugging, week 10 is very chaotic due to presentations and meetings so there will probably not be a lot of time in that week to develop a lot. We also aim to have a working version continuously throughout the process of development. There is no budget, the team works free of charge and even pays tuition to be able to do this project.
+There's a time-frame of 10 weeks for the entire project. We aim to finish the product in week 9 so we will have the time to debug in week 10. Apart from the debugging, week 10 is very chaotic due to presentations and meetings so there will probably not be a lot of time in that week to develop much more. We also aim to have a working version continuously throughout the process of development. There is no budget, the team works free of charge and even pays tuition to be able to do this project.
```

Comment:
Volgens de planning is in week 9 de final input for SIG, dan moet dus de code af zijn